templates/attend.html

```diff
@@ -78,37 +79,6 @@ <h3>Accessibility</h3>
             </div>
         </div>
     </div>
-    <div id="getting-there">
-        <div class="content-wrapper">
-            <div class="content blurb-and-image reverse">
-                <img src="/images/mitosis-garden.webp" width="500" height="380" alt="">
-                <div class="col">
-                    <h2>Getting There</h2>
-                    <p>tbd</p>
-                </div>
-            </div>
-            <div class="content blurb-and-image">
-                <div class="col">
-                    <h2>Central Station</h2>
-                    <p>tbd</p>
-                </div>
-                <img src="/images/Hauptbahnhof.webp" width="500" height="380" alt="">
-            </div>
-            <div class="content blurb-and-image reverse">
-                <img src="/images/berlin-brandenburg.webp" width="500" height="380" alt="">
-                <div class="col">
-                    <h2>Airport</h2>
-                    <p>tbd</p>
-                </div>
-            </div>
-        </div>
-    </div>
-    <div id="food-guide">
-        <div class="content-wrapper">
-            <h2>Food guide</h2>
-            tbd
-        </div>
-    </div>
-    </div>
+</div>
```

Comment:
what's wrong with the div indentation in this file? you're adding a 2nd top level div?